go.mod

```diff
@@ -115,19 +113,18 @@ require (
 	github.com/prometheus/procfs v0.8.0 // indirect
 	github.com/r2ishiguro/vrf v0.0.0-20180716233122-192de52975eb // indirect
 	github.com/rcrowley/go-metrics v0.0.0-20200313005456-10cdbea86bc0 // indirect
-	github.com/rogpeppe/go-internal v1.8.1 // indirect
 	github.com/rs/cors v1.8.2 // indirect
 	github.com/sasha-s/go-deadlock v0.2.1-0.20190427202633-1595213edefa // indirect
 	github.com/spf13/afero v1.8.2 // indirect
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/stretchr/objx v0.4.0 // indirect
 	github.com/subosito/gotenv v1.4.1 // indirect
-	github.com/tendermint/tendermint v0.34.14 // indirect
+	github.com/tendermint/tendermint v0.34.20 // indirect
```

Comment:
@tnasu , is this version no problem?

Reply:
No problem. There is nothing that changes between `v0.34.14` and `v0.34.20`. The indirect dependency makes `cosmos/iavl`. It uses for the below:
* ProofOperator: 
  * https://github.com/tendermint/tendermint/blob/v0.34.14/crypto/merkle/proof_op.go#L21-L25
  * https://github.com/tendermint/tendermint/blob/v0.34.20/crypto/merkle/proof_op.go#L21-L25
* ProofOp: 
  * https://github.com/tendermint/tendermint/blob/v0.34.14/proto/tendermint/crypto/proof.proto#L32-L36
  * https://github.com/tendermint/tendermint/blob/v0.34.20/proto/tendermint/crypto/proof.proto#L32-L36